Dockerfile.dev

```diff
@@ -1,9 +1,12 @@
 FROM node:19-buster-slim
 WORKDIR /iris-messenger/
 ENV NODE_OPTIONS=--openssl-legacy-provider
-RUN apt-get update && apt-get install -y python3 build-essential
```

Comment:
Unnecessary to run in dev mode

Reply:
I think this is still needed, otherwise you get

```
#0 25.31 error /iris-messenger/node_modules/bufferutil: Command failed.
#0 25.31 Exit code: 1
#0 25.31 Command: node-gyp-build
#0 25.31 Arguments:
#0 25.31 Directory: /iris-messenger/node_modules/bufferutil
#0 25.31 Output:
#0 25.31 gyp info it worked if it ends with ok
#0 25.31 gyp info using node-gyp@9.3.1
#0 25.31 gyp info using node@19.7.0 | linux | arm64
#0 25.31 gyp ERR! find Python
#0 25.31 gyp ERR! find Python Python is not set from command line or npm configuration
#0 25.31 gyp ERR! find Python Python is not set from environment variable PYTHON
#0 25.31 gyp ERR! find Python checking if "python3" can be used
#0 25.31 gyp ERR! find Python - "python3" is not in PATH or produced an error
#0 25.31 gyp ERR! find Python checking if "python" can be used
#0 25.31 gyp ERR! find Python - "python" is not in PATH or produced an error
#0 25.31 gyp ERR! find Python
```

Reply:
I didn't see the error on my side. Odd, but I'll take your word for it.